modules/kinesis/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/KinesisEnrich.scala

```diff
@@ -71,7 +72,8 @@ object KinesisEnrich extends Enrich {
              forceDownload,
              credentials.aws,
              credentials.gcp,
-             Option(awsRegion)
+             Option(awsRegion),
+             new URI("http://todo-s3-endpo")
```

Comment:
there are todo urls here and there, I have to propagate them in the code. due to the poor structure and tech-debt, it is not easy and I don't want put a new duct tape.